Source/WebCore/html/HTMLInputElement.idl

```diff
@@ -112,6 +112,7 @@ enum AutofillButtonType {
     // WebKit extension for autofill.
     [EnabledForWorld=allowAutofill] attribute boolean autofillAvailable;
     [EnabledForWorld=allowAutofill] attribute boolean autofilled;
+    [EnabledForWorld=allowAutofill] readonly attribute boolean lastChangeWasUserEdit;
```

Comment:
It's okay to add this just for autofill but we should probably just expose this as a regular JS API.

Reply:
I think that should be done separately.  That is a change with much broader impacts.